tests/integration/components/sl-chart-test.js

```diff
@@ -0,0 +1,117 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-chart', 'Integration | Component | sl chart', {
+    integration: true
+});
+
+test( 'Defaults applied correctly', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries options=testoptions}}
+      {{/sl-chart}}
+    ` );
```

Comment:
Please have a conversation with @juwara0 and @azizpunjani, and@joshforisha, as https://github.com/softlayer/sl-ember-components/pull/647 introduced the pattern that I directed you to change this PRs changes to, and I assumed that it was the desired/correct one, as it passed the linting.

Within that PR, https://github.com/softlayer/sl-ember-components/blob/EmberCli-1.13.8/tests/integration/components/sl-modal-body-test.js#L9 also introduced an inconsistency within/against itself that I missed that needs to be brought into alignment with whatever is decided.